tests/migrations/test_optimizer.py

```diff
@@ -631,3 +632,34 @@ def test_optimize_through_fields(self):
                 migrations.CreateModel("Bar", [("width", models.IntegerField())]),
             ],
         )
+
+    def test_optimize_elidable_operation(self):
+        self.assertOptimizesTo(
+            [
+                operations.special.ElidableOperation(elidable=True),
```

Comment:
I thought `elidable` operations were only meant to be optimizeable through not always reduced if not necessary for further optimization to take place.